### ISSUE TITLE
Get rid of main office number on contact page ios

### ISSUE THREAD

**Issue by markvoortmanppu** [pointparkuniversity/familyhouse#4]

**Comment by jeffreyjseamanppu** [comment]
Fixed this a while ago.

**Comment by jeffreyjseamanppu** [comment]
I will let you review and close.[Example of the fixed screen](http://familyhouse.it.pointpark.edu/images/ioscontract.png)

**Comment by markvoortmanppu** [comment]
Isn't the main office number still at the top?

**Comment by jeffreyjseamanppu** [comment]
That's right. Remove Main, but keep housing? Is that what it was supposed to be?

**Comment by markvoortmanppu** [comment]
I think so, but not entirely sure.

**Comment by markvoortmanppu** [comment]
#5 is also relevant, it looks like that was changed.
So if we remove Main we should be in a good spot.

**Comment by jeffreyjseamanppu** [comment]
This is now completed. Checked new code in. Please review to ensure it's correct. @markvoortmanppu
